### ISSUE TITLE
Is this repo dead?

### ISSUE THREAD

**Issue by kmcrawford** [aksonov/react-native-router-flux#3846]
I see many PRs & issues for simple things like updating dependancies.  I'm assuming this project has been abandoned.  If so are there maintained forks?

**Comment by nabilfreeman** [comment]
Yes, but the dream is still alive. Check out Expo Router if you want a clean routing system.

**Comment by lnxph-dev-ncalizar** [comment]
@nabilfreeman  is there an alternative to Expo Router for bare react native projects?

**Comment by nabilfreeman** [comment]
Plain `react-navigation` is the only real choice (in fact Expo Router is built on top of it).

So following the instructions here is your best bet:

https://reactnavigation.org/docs/getting-started

It is possible to manage your screens in a declarative way by creating a single stack navigator, declaring all your screens in some sort of config file and looping through it to embed all the screens inside the stack. I personally really hate picking up a project with `Screens.tsx` spaghetti so if you are the same you can rest assured it's manageable with bare RN Navigation.

There are some limitations such as not being able to display modal type screens etc. but you can get pretty far. Also if you need web support where you can refresh the page without losing your props then param based screen names is required which makes setup a bit trickier.

Honestly if you want a clean managed experience it's hard to beat Expo Router. Consider setting up a managed Expo environment and try moving your code in and see if everything still works.
